liferay-tango/liferay-portal
# LPS-144627 Migrate Treeview component in content-dashboard

In this change, we're migrating from our previous
[Treeview](https://github.com/liferay/liferay-portal/blob/c5a9a72d3a58a504233e4cc799652fc9f544ba5e/modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/treeview/Treeview.js) component to the new [ClayTreeView](https://github.com/liferay/clay/blob/d7340f875372ba7ac11752aedd70f4343100241b/packages/clay-core/src/tree-view/TreeView.tsx) component.

**Test plan**:
- Fetch this branch
- Deploy the content-dashboard-web OSGi module
- From the global menu navigate to "Content Dashboard"
- In the "Content" table, click the "Filter and Order" drop down menu
  and select "Type" or "Extension"
- Since this is behind a feature flag, see  [this](https://github.com/liferay/liferay-portal/commit/7ac999cb5d9843782c056d2f30ecf0b8666bcccc) commit for more information
